README.md

```diff
@@ -546,16 +570,16 @@ Adding new files? Get the license header correct with:
 
 Follow the steps below to release a new version of Nancy. You need to be part of the `deploy from circle ci` group for this to work.
 
-  1. Checkout/pull the latest `main` branch, and create a new tag with the desired semantic version and a helpful note:
-  
-         git tag -a v1.0.x -m "Helpful message in tag."
-         
-  2. Push the tag up:
-  
-         git push origin v1.0.x
-         
-  3. There is no step 3.
-          
+1. Checkout/pull the latest `main` branch, and create a new tag with the desired semantic version and a helpful note:
+
+   git tag -a v1.0.x -m "Helpful message in tag."
```

Comment:
While many of the formatting changes are an improvement, such changes do not appear to be related to the intent of the PR (Arch install instructions). Happy to include the formatting changes, as they do no harm, and appear to improve things.

However, these particular changes to the Release Steps appear to remove the commands from "code" blocks. Maybe you could revert this particular format change? (Or indent the commands to they remain "code" blocks?)

Reply:
Good point, @bhamail. Thanks! It's fixed now.